lib/lighthouse/benefits_documents/configuration.rb

```diff
@@ -0,0 +1,133 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/rest'
+require 'lighthouse/auth/client_credentials/jwt_generator'
+require 'lighthouse/auth/client_credentials/service'
+
+module BenefitsDocuments
+  ##
+  # HTTP client configuration for the {BenefitsClaims::Service},
+  # sets the base path, the base request headers, and a service name for breakers and metrics.
+  #
+  class Configuration < Common::Client::Configuration::REST
+    self.read_timeout = Settings.lighthouse.benefits_documents.timeout || 20
+
+    SYSTEM_NAME = 'vets.gov'
+    API_SCOPES = %w[system/documents.read system/documents.write].freeze
```

Comment:
On the [Benefits Documents CCG page](https://developer.va.gov/explore/authorization/docs/client-credentials?api=benefits-documents) the scopes are listed as just `document.read and documents.write`. I know there's been some changes to the scope names for benefit claims. 

Would be good to double check these with the lighthouse team.

Reply:
+1

Reply:
Updated